### DISCUSSION TITLE
Release plan for doctrine/orm 2.10.3

### DISCUSSION THREAD

**Discussion by armenio** [doctrine/doctrine2#9223] [Support Questions] [answered]
What is the release plan for doctrine/orm 2.10.3 or higher?

**Comment by derrabus** [comment, marked as answer]
There is none yet. Do you urgently need a new 2.10 release?

**Reply by armenio** [reply]
Yes, I have issues with the bug fixed in this PR: https://github.com/doctrine/orm/pull/9153
So I'm still using v2.9.5 until release v2.10.13 or higher

**Reply by derrabus** [reply]
2.10.3 is on its way now. Sorry that you had to wait that long.

**Reply by armenio** [reply]
Wow.
thank you very much for your attention
